examples/README.md

```diff
@@ -167,25 +174,35 @@ RewriteRule ^ https://example.com/path/file.php?%1? [R=302,L]
 
 In this example, the *CondPattern* `(.*)` in *RewriteCond* will match every characters in the query string (`a=1&b=2`) and put it in group one. This group one (`%1`) is later used in the *Substitution* of *RewriteRule*.
 
-
-
 Note that the special character to recall groups from *CondPattern* of *RewriteCond* is `%` (unlike the special character to recall groups from *Pattern* of *RewriteRule*, which is `$`).
 
 The character `?` at the end of the *Substitution* of *RewriteRule* tells the server not to pass the query string to the final URL after rewrite.
 
 
-## README.md
+### Example 4: Publish vocabularies with W3ID
 
-Each ID hosted on W3ID is expected to have a file named `README.md` containing an information about the ID itself and an information about the maintainer(s). This can be more elaborate than the information inside `.htaccess`.
+If you plan to publish a vocabulary/ontology with W3ID,
+see [/example/.htaccess](https://github.com/perma-id/w3id.org/blob/master/example/.htaccess)
+and https://w3id.org/example/.
 
-The `.md` file extension at the end of the file name indicates that the file uses Markdown markup language for text formatting. You can have tables and images as well, if needed.
 
-GitHub will automatically display the content of a `README.md` to repository visitors.
+### Example 5: Version-aware URIs of ontologies
 
-An example of a good README file: [w3id.org/dggs/README.md](https://github.com/perma-id/w3id.org/blob/master/dggs/README.md)
+See [/OWLunit/.htaccess](https://github.com/perma-id/w3id.org/blob/master/OWLunit/.htaccess).
 
 
-## Publish vocabularies with W3ID
+## README.md
 
-If you plan to publish a vocabulary/ontology with W3ID,
-see https://w3id.org/example/.
+Each ID hosted on W3ID is required to have a file named `README.md` containing
+an information about the ID itself and an information about the maintainer(s).
+This can be more elaborate than the information inside `.htaccess`.
+
+The `.md` file extension at the end of the file name indicates that the file
+uses Markdown markup language for text formatting. You can have tables and
+images as well, if needed.
+
+GitHub will automatically display the content of a `README.md` to repository
+visitors.
+
+An example of a good README file:
+[w3id.org/dggs/README.md](https://github.com/perma-id/w3id.org/blob/master/dggs/README.md)
```

Comment:
```suggestion
Each ID hosted on W3ID is required to have a file named `README.md` containing
information about the ID itself and information about the maintainer(s).
This can be more elaborate than the information inside `.htaccess`.

The `.md` file extension at the end of the file name indicates that the file
uses Markdown markup language for text formatting. You can have tables and
images as well, if needed.

GitHub will automatically display the content of a `README.md` to repository
visitors.

An example of a good README file:
[`w3id.org/dggs/README.md`](https://github.com/perma-id/w3id.org/blob/master/dggs/README.md)
```